plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/properties/PropertyEditorUtils.java

```diff
@@ -116,6 +118,8 @@ public static CellEditor createCellEditor(Composite parent, Object object, DBPPr
                 parent,
                 strings,
                 SWT.DROP_DOWN | SWT.READ_ONLY);
+        } else if (Timestamp.class.isAssignableFrom(propertyType)) {
```

Comment:
Not the best idea to use JDBC specific data type in generic control.
Can we use just Date or LocalDate?